core/src/main/java/org/jboss/pnc/rex/rest/InternalEndpointImpl.java

```diff
@@ -59,22 +59,22 @@ public void finish(String taskName, FinishRequest result) {
     @Override
     @Retry
     @Fallback(fallbackMethod = "objectFallback", applyOn = {RollbackException.class, ArcUndeclaredThrowableException.class})
-    @RolesAllowed("user")
+    @RolesAllowed({ "pnc-users-rex-admin", "pnc-users-admin" })
     public void succeed(String taskName, Object result) {
         taskProvider.acceptRemoteResponse(taskName, true, result);
     }
 
     @Override
     @Retry
     @Fallback(fallbackMethod = "objectFallback", applyOn = {RollbackException.class, ArcUndeclaredThrowableException.class})
-    @RolesAllowed("user")
+    @RolesAllowed({ "pnc-users-rex-admin", "pnc-users-admin" })
```

Comment:
Leaving out the naming: `"pnc-users-rex-admin", "pnc-users-rex-user", "pnc-users-admin"`